roles/idrac_server_powerstate/molecule/pushpowerbutton/converge.yml

```diff
@@ -2,6 +2,8 @@
 - name: Converge
   hosts: all
   gather_facts: false
+  vars:
+    wait_seconds: 180
```

Comment:
To keep it in sync can we rename this fact and register variables to role_name_

Reply:
Sure